osgar/drivers/logsocket.py

```diff
@@ -175,18 +175,29 @@ def _send(self, data):
 
 class LogHTTP:
     def __init__(self, config, bus):
-        bus.register('raw')
-        self.input_thread = Thread(target=self.run_input, daemon=True)
+        bus.register('raw', 'response')
+        self.url = config.get('url')
+        self.output = config.get('output', False)
```

Comment:
is not `url` and `output` completely independent? If yes then they do not have to be in one common module or what is the motivation to keep it together?